pkg/diagnose/connectivity/endpoint_info.go

```diff
@@ -47,6 +49,15 @@ var (
 	seriesEndpointInfo       = endpointInfo{endpoints.SeriesEndpoint, "POST", emptyPayload}
 	sketchSeriesEndpointInfo = endpointInfo{endpoints.SketchSeriesEndpoint, "POST", emptyPayload}
 
+	// Flare endpoint
+	flareEndpointInfo = endpointInfo{transaction.Endpoint{Route: createFlareEndpoint(), Name: "flare"}, "HEAD", nil}
+
 	endpointsInfo = []endpointInfo{v1SeriesEndpointInfo, v1CheckRunsEndpointInfo, v1MetadataEndpointInfo, v1IntakeEndpointInfo,
-		seriesEndpointInfo, sketchSeriesEndpointInfo, v1ValidateEndpointInfo}
+		seriesEndpointInfo, sketchSeriesEndpointInfo, v1ValidateEndpointInfo, flareEndpointInfo}
 )
+
+func createFlareEndpoint() string {
+	// Create flare endpoint to the shape of "https://<version>-flare.agent.datadoghq.com/support/flare"
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏nice comment, makes it much clearer